content/rainstorms/2_epp.md

```diff
@@ -0,0 +1,12 @@
+---
+image: rainstorms/epp_sepp.png
+headline: EPP and SEPP
```

Comment:
```suggestion
image: rainstorms/epp_sepp.png
headline: Precipitation metrics
```

Reply:
Thanks for the comment. It is changed now.